schemas/context/profile-work-details.schema.json

```diff
@@ -17,27 +17,75 @@
     "profile-work-details": {
       "properties": {
         "xdm:workAddress": {
-          "title": "Work address",
+          "title": "Work Address",
           "$ref": "https://ns.adobe.com/xdm/common/address",
           "description": "A work postal address."
         },
+        "xdm:otherAddress": {
```

Comment:
Same as Other Phone. Include this there and not in work details.

Reply:
Even if we create a new data type, it would still have the attributes of an address which are already existing in the "https://ns.adobe.com/xdm/context/address" data type.  Would creating another object as I did and just referring the same datatype cause any issues ? I just want to understand this a little bit more. Please clarify.